### ISSUE TITLE
Raid up-scaling is crazy

### ISSUE THREAD

**Issue by ifeelrobbed** [ph1lt0r/PA-WoW-Server#10]
**Describe the bug**
Raid boss goes HP up over 3x when leveling from 70 to 74.

**To Reproduce**
1. Go to raid at appropriate level
2. Level a little
3. Go back to raid
4. See HP of boss
5. Nope out because the fight already took 20+ minutes the first time around

**Expected behavior**
Retail does not scale up at all. At the very least it shouldn't be more difficult if you level up.  Should be easier for sure.

**Screenshots**
![unknown](https://user-images.githubusercontent.com/67766369/87361374-b8a34a80-c531-11ea-9967-e7488aff5c66.png)
![unknown-1](https://user-images.githubusercontent.com/67766369/87361382-be009500-c531-11ea-916c-bd6f2fbefe35.png)

**Additional context**
Phil is dreamy

**Comment by ifeelrobbed** [comment]
It should be noted that my damage capability is nowhere near 3x to match. 
70 Shadowbolt crit for 2.2 to 2.5k
74 crit for 3k
I'm in good Northrend instance blues at the moment.

**Comment by LukeArmstrongkc** [comment]
I agree that the scaling seems to be too severe.

**Comment by DrWeggs** [comment]
Confirming wonky scaling. At lvl 80. Just tried the Nexus (low70), and the trash mobs had 30k hp. Couldn’t do it. Went into the Oculus (upper70) and the trash had 3k hit points.

**Comment by ifeelrobbed** [comment]
Checking back at 80
![image](https://user-images.githubusercontent.com/67766369/88058314-e3406500-cb28-11ea-842e-0223b8ffc38f.png)

**Comment by ifeelrobbed** [comment]
For kicks I engaged him. 25 minutes. Half of it was waiting for mana to tick high enough to cast a spell. 

I had food buffs, elixer, a Gift of the Wild from some thing I found on the AH, all buffs, all cooldowns popped, wearing all purples except for 2 slots. Gear score 4683.

Edit:
The challenge was pretty good. It had my full attention the entire time, but lasted waaaaaaaaaaaaaaaaaaaaaaay too long.
